### DISCUSSION TITLE
Deploy Hook creates a job, but no build is created

### DISCUSSION THREAD

**Discussion by psantosTMI** [vercel/community#1750] [Help]
### Summary

I'm triggering the Deploy hook manually and via webhook. Getting a positive response for the job:

{
  "job": {
    "id": "---------",
    "state": "PENDING",
    "createdAt": 1678468885002
  }
}

But no build is being built in my deployments. It is building, however, when I make a commit via git. Deployments on my non-main branches also seem to be working.

**Comment by Tobi-mmt** [comment]
Have you enabled a "Ignored Build Step" in your project under "settings" => "git" => "Ignored Build Step" ?
If you decribed here that you don't want a rebuild if the code did not change, than the hook does not have any effect, since the code did not change, only the content.
